### PR TITLE
Feature/484 realtime mi

### PR DESCRIPTION
**Pull request by kennyevil**
## Changes in this PR

Add an admin page for user analysts and agents to view real time statistics on cases in Case Management and download a CSV with the information

## Screen-shots or screen-capture of UI changes
Screenshot 2022-02-28 at 17-36-44 Case statistics

### CODE REVIEW

**Comment by kennyevil** [comment]
[#484 Real-time MI for Proc Ops Managers (short-term solution)](https://trello.com/c/l6whRQwy/365-484-real-time-mi-for-proc-ops-managers-short-term-solution)

File: spec/features/support/admin/show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      visit "/support/admin"
+    end
+
+    it "shows the page content" do
+      expect(page).to have_title "Case statistics"
+      expect(find("h1.govuk-heading-l")).to have_text "Case statistics"
+      expect(all("th.govuk-table__header")[0]).to have_text "Number of cases"
+      expect(all("td.govuk-table__cell")[0]).to have_text "5"
+
+      expect(all("h2.govuk-heading-m")[0]).to have_text "Number of cases by state"
```
Comment (peterdavidhamilton) [inline spec/features/support/admin/show_spec.rb:25]: Pop these in a couple of let blocks

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 2022_02_23_124157) do
+=======
```
Comment (peterdavidhamilton) [inline db/schema.rb:15]: git diff

Comment (kennyevil) [reply db/schema.rb:15]: Gah! I forgot to hit save before I committed!

File: app/controllers/support/admin_controller.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      role =  if current_user.analyst?
+                "analyst"
+              else
+                "agent"
+              end
+
+      Rollbar.info("User role has been granted access.", role: role, path: request.path)
+    end
+
+    def download_cases
```
Comment (ryantk) [inline app/controllers/support/admin_controller.rb:17]: Can you merge this action in with show and it be a .csv response to show?

**Review by d-a-v-e — APPROVED** [review]

**Review by ryantk — APPROVED** [review]

**Pull request merged**